setup.py

```diff
@@ -78,5 +78,7 @@ def find_version(*file_paths):
         'Programming Language :: Python',
         'Topic :: Software Development :: Libraries :: Python Modules'
     ],
-
+      entry_points=dict(console_scripts=[
+          'wofpy_config = wof.wofpy_config:main']
```

Comment:
This entry point should be available in the user path. I will add a test for it once #65 is merged.